contracts/upgradeable_contracts/amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol

```diff
@@ -0,0 +1,188 @@
+pragma solidity 0.4.24;
+
+import "../../interfaces/IAMB.sol";
+import "../Ownable.sol";
+import "openzeppelin-solidity/contracts/AddressUtils.sol";
+import "../Initializable.sol";
+import "../ERC677Bridge.sol";
+import "../BaseOverdrawManagement.sol";
+import "../ReentrancyGuard.sol";
+import "../Upgradeable.sol";
+import "../Claimable.sol";
+import "../VersionableBridge.sol";
+
+contract BasicAMBErc677ToErc677 is
+    Initializable,
+    Ownable,
+    ReentrancyGuard,
+    Upgradeable,
+    Claimable,
+    VersionableBridge,
+    BaseOverdrawManagement,
+    ERC677Bridge
+{
+    bytes32 internal constant BRIDGE_CONTRACT = keccak256(abi.encodePacked("bridgeContract"));
+    bytes32 internal constant MEDIATOR_CONTRACT = keccak256(abi.encodePacked("mediatorContract"));
+    bytes32 internal constant REQUEST_GAS_LIMIT = keccak256(abi.encodePacked("requestGasLimit"));
+
+    function initialize(
```

Comment:
I think for consistency it will be nice to apply those changes for this mode too. I'll update the branch with the changes from `develop` and apply the changes

Reply:
- Added decimal shift feature https://github.com/poanetwork/poa-bridge-contracts/pull/273/commits/1321c1f38ce5f855c6422e0c024af5978e3ee457
- Combined parameters into arrays https://github.com/poanetwork/poa-bridge-contracts/pull/273/commits/4368902924b2d737cc8c896cbf695daa7f68615d
- Updated deploy script with both changes https://github.com/poanetwork/poa-bridge-contracts/pull/273/commits/0d7c5a5eed9ef4d9fc064188f78bb61561990c2d